course02/homework00/joost/vector.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include <array>
+#include <stdexcept>
+
+class VectorInt
+{
+	public:
```

Comment:
coding style: access specifiers are not indented, rest of the class should only be indented with one level, not two